crates/trace-rpc-types/Cargo.toml

```diff
@@ -0,0 +1,48 @@
+[package]
+name = "alloy-trace-rpc-types"
```

Comment:
this should be called alloy-rpc-trace-types

and the folder name rpc-trace-types